apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipant/Dto/UserAuditLogsDto.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+
+namespace Energinet.DataHub.WebApi.Controllers.MarketParticipant.Dto
+{
+    public sealed record UserAuditLogsDto(IEnumerable<UserRoleAssignmentAuditLogDto> RoleAssignments);
```

Comment:
nitpick: variable name should probably include Audit somewhere

Reply:
Fixed.